src/PointwiseFunctions/AnalyticData/GrMhd/MagneticRotor.hpp

```diff
@@ -0,0 +1,229 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <limits>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Minkowski.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/IdealFluid.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/Hydro/Tags.hpp"
+#include "Utilities/MakeArray.hpp"  // IWYU pragma: keep
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+// IWYU pragma:  no_include <pup.h>
+
+/// \cond
+namespace PUP {
+class er;  // IWYU pragma: keep
+}  // namespace PUP
+/// \endcond
+
+namespace grmhd {
+namespace AnalyticData {
+
+/*!
+ * \brief Analytic initial data for a magnetic rotor.
+ *
+ * This is a test first described in \cite Balsara1999 for classical MHD and
+ * later generalised to relativistic MHD in \cite DelZanna2002rv
+ *
+ * This effectively 2D test initially consists of an infinitely long cylinder of
+ * radius `RotorRadius` rotating about the z-axis with the given
+ * `AngularVelocity`. The rest mass density of the fluid inside the rotor,
+ * `RotorDensity`, is higher than the `BackgroundDensity` outside of the rotor.
+ * The fluid is at a constant `Pressure`.  The rotor is embedded in a constant
+ * `MagneticField` (usually taken to be along the x-axis).  The fluid is an
+ * ideal fluid with the given `AdiabaticIndex`.  Evolving the initial data,
+ * magnetic braking will slow down the rotor, while dragging the magnetic field
+ * lines.
+ *
+ * The standard test setup is done on a unit cube with the following values
```

Comment:
Is there a known timescale associated with the magnetic braking for these quantities? It would be useful for someone running a test to know what to expect at a certain code time, at least for these inputs, if known